src/acvp_build_register.c

```diff
@@ -900,12 +900,16 @@ static ACVP_RESULT acvp_build_rsa_sig_register_cap(JSON_Object *cap_obj, ACVP_CA
         if (!rsa_cap_mode) {
             return ACVP_MISSING_ARG;
         }
+        result = acvp_lookup_prereqVals(cap_obj, cap_entry);
+        if (result != ACVP_SUCCESS) { return result; }
     } else if (cap_entry->cipher == ACVP_RSA_SIGVER) {
         json_object_set_string(cap_obj, "mode", "sigVer");
         rsa_cap_mode = cap_entry->cap.rsa_sigver_cap;
         if (!rsa_cap_mode) {
             return ACVP_MISSING_ARG;
         }
+        result = acvp_lookup_prereqVals(cap_obj, cap_entry);
+        if (result != ACVP_SUCCESS) { return result; }
```

Comment:
Style guide prohibits single line 'if' statements.
```suggestion
        if (result != ACVP_SUCCESS) {
            return result;
        }
```